### ISSUE TITLE
Get Latitude and Longitude for each State/county pair

### ISSUE THREAD

**Issue by banjtheman** [banjtheman/defundthepolice#24]
# What is the Task
Create a CSV with the following information

```
state,county,latidue,longitude 
```

## Why do we want to do this
In order to create maps, we will need the latitude and longitude coordinates for each state/county

## How can I get started?
1. Collect a list of states/county (https://en.wikipedia.org/wiki/List_of_counties_by_U.S._state_and_territory)
2. Use a forward geocoding service like https://docs.mapbox.com/api/search/#geocoding to get the coordinates 
3. Store information in a CSV file

## Definition of Done
CSV produced with all the information

**Comment by sarthak55k** [comment]
can i do this?

**Comment by sarthak55k** [comment]
How to get the list of all countries?

**Comment by banjtheman** [comment]
> 
> 
> How to get the list of all countries?

Up to you, you can scrape, use an API, or whatever you are comfortable with

**Comment by chandantaneja** [comment]
I would like to work on this issue.

**Comment by cupOJoseph** [comment]
@chandantaneja or @sarthak55k: check out https://www.census.gov/library/publications/2011/compendia/usa-counties-2011/file-layout.html#part01

You can download a spreadsheet from the census with a list of counties already setup. You will have to delete some columns but that shouldnt be too hard.

**Comment by chandantaneja** [comment]
@jschiarizzi We need Lat and Long of all the states of a particular country or just the country itself ?

**Comment by banjtheman** [comment]
Should be for county level
```
state,county,latidue,longitude 
VA,Arlington,Arlington_latidue,Arlington_longitude 
```

**Comment by Medha08** [comment]
Can I work on this issue?

**Comment by callumrollo** [comment]
Hi @banjtheman, I've had a go at this and I think I have the data you need

![image](https://user-images.githubusercontent.com/28703282/95221619-88491080-07ef-11eb-9463-ff43f29d5e22.png)


Shall I do a PR?

**Comment by banjtheman** [comment]
> 
> 
> Hi @banjtheman, I've had a go at this and I think I have the data you need
> 
> ![image](https://user-images.githubusercontent.com/28703282/95221619-88491080-07ef-11eb-9463-ff43f29d5e22.png)
> 
> Shall I do a PR?

yes of course

**Comment by banjtheman** [comment]
closed by #54
